### ISSUE TITLE
Contributing?

### ISSUE THREAD

**Issue by dschonholtz** [Blealtan/efficient-kan#5]
Hey I have a fork, with some not useful stuff on it yet. (Mostly just profiling showing forward passes suck due to b-splines and some comparisons to MLPs.)

Do you want folks to contribute to this?
Are you interested in making b-splines more efficient with something like: https://github.com/GistNoesis/FourierKAN/blob/main/fftKAN.py

Let me know what you think.

**Comment by sidhu2690** [comment]
Hi there, 
I am interested in contributing!

**Comment by dschonholtz** [comment]
Hey @Blealtan I am gonna roll my own version of this repo since it seems like you aren't actively replying. 
I'll be pulling in the conv kan, fourier kan and doing comparisons on all of them.

Also, hoping to get into really doing break downs of why things are happening the way they are happening.

**Comment by minhhnguyen0312** [comment]
Hi there @dschonholtz. Have you tried to measure the performance of ur algorithm compare to this one and the original?

You may refer to this [tutorial](https://github.com/KindXiaoming/pykan/blob/master/tutorials/Example_7_continual_learning.ipynb)

I looking forward to your results. Have a nice day!

**Comment by Blealtan** [comment]
Yeah the Fourier variant definitely sounds attractive.

I do have little time maintaining this right now (working on my PhD viva).
Also, this repo mostly serves as a playground to show the idea that KAN (in its current shape) can be reformulated into sth like MLP and become computationally efficient.
If you are going down on real experiments and have performance issues around all of this, please let me know and I'm willing to help.

In short: for now, I don't think I have the time to accept contributions; please go ahead fork the repo or copy code snippets under MIT license.
